cmake/dependencies/ABLASTR.cmake

```diff
@@ -71,6 +80,13 @@ macro(find_ablastr)
         set(WarpX_MPI_THREAD_MULTIPLE ${ImpactX_MPI_THREAD_MULTIPLE} CACHE INTERNAL "" FORCE)
         set(WarpX_IPO ${ImpactX_IPO} CACHE INTERNAL "" FORCE)
 
+        # control FFT cache (option 2)
+        #if(ImpactX_FFT)
+        #   set(AMReX_FFT ON CACHE INTERNAL "")
+        #else()
+        #   set(AMReX_FFT OFF CACHE INTERNAL "")
+        #endif()
+
```

Comment:
This is a commented duplicate of 40-44, because I wasn't sure of the best place to put this.